frontend/packages/dev-console/src/components/project-access/ProjectAccess.tsx

```diff
@@ -131,7 +138,7 @@ const ProjectAccess: React.FC<ProjectAccessProps> = ({ formName, namespace, role
             onReset={handleReset}
             validationSchema={validationSchema}
           >
-            {(formikProps) => <ProjectAccessForm {...formikProps} />}
+            {(formikProps) => <ProjectAccessForm {...formikProps} roles={roles} />}
```

Comment:
Based on the changed hook signature you need extract the roles here:

```suggestion
            {(formikProps) => <ProjectAccessForm {...formikProps} roles={roles.data} />}
```